pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2851,6 +2851,22 @@ let
         };
       };
 
+      ms-python.flake8 = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "flake8";
+          publisher = "ms-python";
+          version = "2023.6.0";
+          hash = "sha256-Hk7rioPvrxV0zMbwdighBAlGZ43rN4DLztTyiHqO6o4";
+        };
+        meta = with lib; {
```

```suggestion
        meta = {
```
Avoid nested `with` whenever possible.

---

Consider reading the conventions for VSCode extensions.
Relevant points:
- changelog
- description
- [nixfmt-rfc-style](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=nixfmt-rfc-style)
https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vscode/extensions/README.md